src/Workspaces/Remote/Core/SolutionAssetStorage.Scope.cs

```diff
@@ -97,10 +100,10 @@ public async ValueTask<object> GetAssetAsync(Checksum checksum, CancellationToke
             {
                 Contract.ThrowIfTrue(checksum == Checksum.Null);
 
-                using var checksumPool = Creator.CreateChecksumSet(checksum);
                 using var _ = Creator.CreateResultMap(out var resultPool);
 
-                await scope.FindAssetsAsync(AssetPath.FullLookupForTesting, checksumPool.Object, resultPool, cancellationToken).ConfigureAwait(false);
+                var checksums = new ReadOnlyMemory<Checksum>([checksum]);
+                await scope.AddAssetsAsync(AssetPath.FullLookupForTesting, checksums, resultPool, cancellationToken).ConfigureAwait(false);
```

Comment:
>AddAssetsAsync [](http://example.com/codeflow?start=28&length=14)

Why the switch to AddAssetsAsync here?

Reply:
It's a test call and was cleaner. Just goes through the to level path without having to make a set and make a lambda.